timebinmgr.c

```diff
@@ -0,0 +1,89 @@
+#include "timebinmgr.h"
+#include <math.h>
+#include "allvars.h"
+
+/*Each integer time stores in the first 10 bits the snapshot number.
+ * Then the rest of the bits are the standard integer timeline,
+ * which should be a power-of-two hierarchy.*/
+
+unsigned int out_from_ti(unsigned int ti)
```

Comment:
I have been in general trying to put the type of return value to a separate line in other codes as I touch the functions. (I think most GNU code does that)

Could you also do this in the new code?